tests/CribblyBackend.Core.UnitTests/Players/Services/PlayerServiceTests.cs

```diff
@@ -0,0 +1,153 @@
+using System.Threading.Tasks;
+using CribblyBackend.Core.Players.Services;
+using CribblyBackend.Test.Support;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using System;
+using CribblyBackend.Core.Players.Repositories;
+using CribblyBackend.Core.Players.Models;
+
+namespace CribblyBackend.Core.UnitTests.Players.Services
+{
+    public class PlayerServiceTests
+    {
+        private readonly IPlayerService _playerService;
+        private readonly IPlayerRepository _fakePlayerRepository;
```

Comment:
I should type this as a `FakePlayerRepository` so it's clear that we're using the fake for these tests. Also, if the `FakePlayerRepository` ever had methods on it that don't exist in `IPlayerRepository`, we'd probably want them.